css/index.css

```diff
@@ -84,12 +84,27 @@ p {
     padding: 0 1%;
     padding-bottom: 20px;  
 }
+.top-content .text-about {
+    width: 100%;
+    padding: 0 1%;
+    padding-top: 25px;
+    padding-bottom: 20px;  
+}
 
 .middle-content {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-evenly;
```

Comment:
This block of code in this CSS selector is missing just *one* line. What is it? This line will take your headline and div of boxes from being side-by-side to being on top of each other.